metron-deployment/roles/elasticsearch/tasks/create-indices.yml

```diff
@@ -32,7 +35,6 @@
   until: result.content.find("green") != -1
   retries: 10
   delay: 60
-  run_once: yes
 
```

Comment:
I had to change this code because we now need to start Elasticsearch, create the indices, stop Elasticsearch, then continue on.  We can't create indices with ES off.  There is a similar need when loading the Geo data in MySQL.